<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XYScatter</class>
 <widget class="QWidget" name="XYScatter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataFrames</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="DQFXYListView" name="treeView_datasets"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="comboBox_lineStyle">
           <item>
            <property name="text">
             <string>line</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>points</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_alpha">
           <property name="text">
            <string>Scatter Alpha</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider_alpha">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_alphaValue">
           <property name="text">
            <string>255</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Legend">
         <property name="title">
          <string>Legend</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox_legend_dfname">
            <property name="text">
             <string>DF Name</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_legend_label">
            <property name="text">
             <string>Item Label</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_legend_ycolumn">
            <property name="text">
             <string>Y-Column</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_plotOptions">
         <item row="0" column="0">
          <widget class="QLabel" name="label_xlabel">
           <property name="text">
            <string>X-Label</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_xlabel"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_ylabel">
           <property name="text">
            <string>Y-Label</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_ylabel"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="PlotWidget" name="plotWidget"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>dataquick.qt.plotwidget</header>
  </customwidget>
  <customwidget>
   <class>DQFXYListView</class>
   <extends>QTreeView</extends>
   <header>dataquick.qt.plotlist</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
